<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRules</class>
 <widget class="QDialog" name="FormRules">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Правила</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>401</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>Список правил</string>
   </property>
   <widget class="QTableView" name="tvRules">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>451</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_confirm">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>480</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_addedit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>480</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>751</width>
     <height>451</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Транзитивность</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>661</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Есть</string>
     </property>
     <widget class="QComboBox" name="cb_C">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_B">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_BC">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>70</y>
        <width>171</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_A">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>AB</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>BC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>30</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AB_direction">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_BC_direction">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_quant_BC">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>40</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_AB">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>30</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>661</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Будет</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>AC</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AC">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>191</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AC_direction">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_quant_AC">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="l_done1">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>400</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;Успешно!&quot;&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Успешно!&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Новый концепт</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>631</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>Есть</string>
     </property>
     <widget class="QComboBox" name="cb_C_2">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>40</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_B_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_BC_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>60</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_A_2">
      <property name="geometry">
       <rect>
        <x>18</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;A&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;B&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BC</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>20</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;C&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_BC_direction_2">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>AB</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AB_direction_2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="cb_AB_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_AB_2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>91</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_BC_2">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>230</y>
       <width>631</width>
       <height>161</height>
      </rect>
     </property>
     <property name="title">
      <string>Будет</string>
     </property>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>46</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;D&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_D">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>130</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_AD_direction">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>AD</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_DC_direction">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>30</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DC</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_DC">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>60</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_AD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>101</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_DC">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_dcON">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_adON">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="cb_bdON">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>20</y>
        <width>31</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_quant_BD">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BD</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_BD">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>70</y>
        <width>181</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cb_BD_direction">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>40</y>
        <width>51</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>---&gt;</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>&lt;---</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QLabel" name="l_done2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>400</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;Успешно!&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#000000;&quot;&gt;Успешно!&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_edit2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>400</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
